README.md

```diff
@@ -70,7 +70,7 @@ To use dotenv for managing environment variables, follow these steps:
     if parent_dir not in sys.path:
```

Comment:
Thanks for pointing this out. I think I either messed up some sequence in the documentation here, or I forgot to remove this.
This code snippet is not supposed to be here. 
Let's not merge this right away. 
I'll raise a new PR with some other suggestions.